docs/codeql/codeql-language-guides/functions-in-cpp.rst

```diff
@@ -58,13 +56,11 @@ You can modify the query to remove functions where a function pointer is used to
      and not exists(FunctionAccess fa | fa.getTarget() = f)
    select f, "This function is never called, or referenced with a function pointer."
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/1505890446605/>`__
-
 This query returns fewer results. However, if you examine the results then you can probably still find potential refinements.
 
-For example, there is a more complicated LGTM `query <https://lgtm.com/rules/2152580467/>`__ that finds unused static functions. To see the code for this query, click **Open in query console** at the top of the page.
+For example, there is a more complicated standard query, `Unused static function <https://codeql.github.com/codeql-query-help/cpp/cpp-unused-static-function/>`__, that finds unused static functions.
 
-   You can explore the definition of an element in the standard libraries and see what predicates are available. Use the keyboard **F3** button to open the definition of any element. Alternatively, hover over the element and click **Jump to definition** in the tooltip displayed. The library file is opened in a new tab with the definition highlighted.
+   You can explore the definition of an element in the standard libraries and see what predicates are available. Highlight the element, right-click to display the context menu, and click **Go to Definition**. The library file is opened in a new tab with the definition of the element highlighted.
```

Comment:
"Highlight the element" - is this necessary? Can't you just right-click it? 
How do you highlight it? (Sorry, I'm speaking without knowledge here. This may be blinding obvious, in which case ignore me. 😄) 

```suggestion
   You can explore the definition of an element in the standard libraries and see what predicates are available. Highlight the element, right-click it to display the context menu, and click **Go to Definition**. The library file is opened in a new tab with the definition of the element highlighted.
```

Reply:
Good point. I'll check.

Reply:
You're right - thanks 💖 

```suggestion
   You can explore the definition of an element in the standard libraries and see what predicates are available. Right-click the element to display the context menu, and click **Go to Definition**. The library file is opened in a new tab with the definition of the element highlighted.
```